Mendelugram/MendelugramTests/ServiceTests/PhotosServiceTest.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PhotosServiceTest.swift
+//  MendelugramTests
+//
+//  Created by David Aldorf on 28/04/2019.
+//
+
+import XCTest
+@testable import Mendelugram
+
+class PhotosServiceTest: XCTestCase {
+
+    // MARK: - Test Cases
+    func testFetchPhotosSuccess() {
+        // Test fetch - impossible with current implementation
+        XCTFail()
```

Comment:
What is a point of this failing test?

Reply:
I just wanted to show file structure. The test case itself could be deleted :) Should I delete the test case?

Reply:
Show to them whatever you find important or interesting :-) But I think that the final solution should have failing test cases. So yes, please delete it or comment out this lane and replace it with a success.